tools/topology/sof-bdw-codec.m4

```diff
@@ -96,3 +95,13 @@ DAI_CONFIG(SSP, 0, 0, Codec,
 		      SSP_CLOCK(fsync, 48000, codec_slave),
 		      SSP_TDM(2, 25, 3, 3),
 		      SSP_CONFIG_DATA(SSP, 0, 24)))
+
+dnl CODEC is defined and will be expanded, need to undefine it before use
+
+ifelse(CODEC, `RT5677',
```

Comment:
@xiulipan there is no need to make the virtual widgets conditional at all. As long as they have unique IDs, no harm in just adding all of them

Reply:
@ranj063 Even if there is no need to add that virtual widgets?